op-chain-ops/genesis/layer_two.go

```diff
@@ -68,6 +69,13 @@ func BuildL2Genesis(config *DeployConfig, l1StartBlock *types.Block) (*core.Gene
 			db.CreateAccount(codeAddr)
 			db.SetState(addr, ImplementationSlot, eth.AddressAsLeftPaddedHash(codeAddr))
 			log.Info("Set proxy", "name", name, "address", addr, "implementation", codeAddr)
+
```

Comment:
You've catched this points! You are so great. Thank you very much. 🙇🏼‍♂️

FiatTokenV2 use the https://github.com/tokamak-network/tokamak-thanos/blob/79ee6926b86e975101ba5b0d71d774ab224b186c/packages/tokamak/contracts-bedrock/src/tokamak-contracts/USDC/L2/tokamak-USDC/upgradeability/UpgradeabilityProxy.sol#L48

Reply:
Thank you very much, Steven

Reply:
Let me create a variable

Reply:
I updated!